vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -16068,12 +16128,14 @@ quantitykind:SpecificOpticalRotatoryPower
 quantitykind:SpecificThrust
   a qudt:QuantityKind ;
   qudt:dbpediaMatch "http://dbpedia.org/resource/Specific_thrust"^^xsd:anyURI ;
+  qudt:hasDimensionVector qkdv:A0E0L1I0M1H0T-2D0 ;
```

Comment:
I'm thinking Length/Time

Reply:
Again, I assumed that specific thrust was a type of thrust w/o thinking, but Wikipedia says it is force/mass flow rate, which agrees with what you said.

Reply:
Agreed.

Reply:
fixed, and removed the skos:broader of quantitykind:Thrust